docs/cubeviz/import_data.rst

```diff
@@ -4,54 +4,91 @@
+will display the collapsed spectrum from ``flux-viewer``.
+
+.. note::
+    Only a single cube can be displayed in an instance of Cubeviz at a given time.
+    To open a second cube, you must first initiate a second instance of Cubeviz.
+
+.. _cubeviz-import-commandline:
+
+Importing data through the Command Line
+---------------------------------------
+
+You can load your data into the Cubeviz application through the command line::
+
+    jdaviz cubeviz /my/directory/cube.fits
+
+.. _cubeviz-import-gui:
+
 Importing data through the GUI
 ------------------------------
 
-The first way you can load your data into the Cubeviz application is
-by clicking the :guilabel:`Import Data` button at the top left of the application's 
-user interface. This opens a dialogue where you can navigate your local
-file system (single click to enter a folder) and select the path of a file 
-that can be parsed as a :class:`~specutils.Spectrum1D`:
-
-.. image:: img/cubeviz_import_data.png
+You can load your data into the Cubeviz application
+by clicking the :guilabel:`Import Data` button at the top left of the application's
+user interface. This opens a dialogue where the user can select a file
+that can be parsed as a :class:`~specutils.Spectrum1D`.
 
-After clicking :guilabel:`Import`, the data file will be parsed and loaded into the 
-application. A green success message will appear if the data import 
-was successful. Afterward, the new data set can be found in the :guilabel:`Data` 
+After clicking :guilabel:`Import`, the data file will be parsed and loaded into the
+application. A notification will appear to let users know if the data import
+was successful. Afterward, the new data set can be found in the :guilabel:`Data`
 tab of each viewer's options menu as described in :ref:`cubeviz-selecting-data`.
 
 .. _cubeviz-import-api:
 
 Importing data via the API
 --------------------------
 
-Alternatively, if you are working in a coding environment like a Jupyter
-notebook, you have access to the Cubeviz helper class API. Using this API,
-you can load data into the application through code using the
-:meth:`~jdaviz.core.helpers.ConfigHelper.load_data`
-method, which takes as input either the name of a local file or a 
-:class:`~specutils.Spectrum1D` object.::
+Alternatively, if users are working in a coding environment like a Jupyter
+notebook, they have access to the Cubeviz helper class API. Using this API,
+users can load data into the application through code using the :meth:`~jdaviz.configs.specviz.helper.Specviz.load_spectrum`
+method, which takes as input a :class:`~specutils.Spectrum1D` object.
 
-    from jdaviz import Cubeviz
+FITS Files
+^^^^^^^^^^
+The example below loads a FITS file into Cubeviz::
+
+    from specutils import Spectrum1D
+    spec3d = Spectrum1D.read("/path/to/data/file")
     cubeviz = Cubeviz()
-    cubeviz.load_data("/Users/demouser/data/cube_file.fits")  # doctest: +SKIP
```

Comment:
doctest does not run on code in the docs unless the code is prepended with a `>>>`, which we are not doing. And as it is now, it gets rendered into the final HTML, which is confusing and ugly. So, I just removed it altogether.

Reply:
Thank you, removing it is fine.